protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -235,12 +294,12 @@ contract Storage {
 }
 
 struct AppStorage {
-    uint8 index; // DEPRECATED - Was the index of the Bean token in the Bean:Eth Uniswap v2 pool, which has been depreciated.
+    uint8 index; // DEPRECATED - Was the index of the Bean token in the BEAN:ETH Uniswap v2 pool, which has been deprecated.
     int8[32] cases; // The 24 Weather cases (array has 32 items, but caseId = 3 (mod 4) are not cases).
     bool paused; // True if Beanstalk is Paused.
     uint128 pausedAt; // The timestamp at which Beanstalk was last paused. 
     Storage.Season season; // The Season storage struct found above.
-    Storage.Contracts c; // DEPRECATED - Previously stored the Contracts State struct. Removed when contract addresses were moved to constants in C.sol.
+    uint256[4] deprecated; // DEPRECATED - Previously stored the Contracts State struct. Removed when contract addresses were moved to constants in C.sol.
```

Comment:
since data is stored in 32 byte chunks, `address[4]`  still used 4 slots of storage, even though the array technically only used (4 * 20) 80 bytes. thus 4 slots are deprecated here (uint256[4] and bytes32[4] are the same in this case)